packages/scan/src/requests/UploadFile.ts

```diff
@@ -0,0 +1,23 @@
+import FormData from 'form-data';
+import { HttpOptions, HttpRequest } from '@secbox/bus';
+
+export interface UploadFileOptions {
+  filename: string;
+  content: string;
+}
+
+export class UploadFile extends HttpRequest<FormData, { id: string }> {
+  constructor(fileOptions: UploadFileOptions, query: Record<string, unknown>) {
+    const payload = new FormData();
+    payload.append('file', fileOptions.content, fileOptions.filename);
```

Comment:
`content-type` is expected here

```suggestion
    payload.append('file', fileOptions.content, { filename: fileOptions.filename, contentType: fileOptions.contentType });
```